src/components/PostsList/PostsList.jsx

```diff
@@ -1,37 +1,57 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
 import './PostsList.scss';
+import { getUserPosts } from '../../api/posts';
 
-export const PostsList = () => (
-  <div className="PostsList">
-    <h2>Posts:</h2>
+export const PostsList = ({ userId, postId, setPostId }) => {
+  const [posts, setPosts] = useState([]);
 
-    <ul className="PostsList__list">
-      <li className="PostsList__item">
-        <div>
-          <b>[User #1]: </b>
-          sunt aut facere repellat provident occaecati excepturi optio
-        </div>
-        <button
-          type="button"
-          className="PostsList__button button"
-        >
-          Close
-        </button>
-      </li>
+  useEffect(() => {
+    getUserPosts(userId)
+      .then(postsOfUser => setPosts(postsOfUser));
```

Comment:
```suggestion
    getUserPosts(userId)
      .then(setPosts);
```
можно немного сократить до такой записи (аналогично в PostDetails)